mne/preprocessing/maxwell.py

```diff
@@ -142,10 +150,20 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
 
         * Double floating point precision
         * Handling of 3D (in additon to 1D) fine calibration files
-        * Processing of data from (un-compensated) non-Elekta systems
+        * **Experimental** processing of data from (un-compensated)
+          non-Elekta systems
         * Automated processing of split (-1.fif) and concatenated files
         * Epoch-based movement compensation as described in [1]_ through
-          :func:`mne.epochs.average_movements`.
+          :func:`mne.epochs.average_movements`
+
+    Use of Maxwell filtering routines with non-Elekta systems is currently
+    **experimental**. Worse results for non-Elekta systems are expected due
+    to (at least):
+
+        * Missing fine-calibration and cross-talk cancellation data for
+          other systems.
+        * Processing with reference sensors has not been vetted.
+        * Regularization of components may not work well for all systems.
```

Comment:
Should we add something about untested integration point configurations since they're presumably not using Abramowitz/Stegun definitions? Something like:

`*Untested geometry of integration points.`